rpcserver.go

```diff
@@ -2981,7 +2981,7 @@ func SignRawTransaction(w *wallet.Wallet, chainSvr *chain.Client, icmd interface
 	// `complete' denotes that we successfully signed all outputs and that
 	// all scripts will run to completion. This is returned as part of the
 	// reply.
-	complete := true
+	var signErrors []btcjson.SignRawTransactionError
```

Comment:
For reference, core does not set this field in the json object.  Leaving it nil will have the same effect since the omitempty struct tag is specified.